### ISSUE TITLE
Bug: JEXL writer.println prints blank line in output after intended line

### ISSUE THREAD

**Issue by SwisscomHopperDevs** [swisscom/JCR-Hopper#27]
Expecting that writer.println() prints its contents followed by newline. In output however it prints the line and then an extra blank line. 

<img width="555" alt="image" src="https://github.com/user-attachments/assets/57883beb-c401-416f-b9ac-782ea12bad39">

Also, expect writer.print() to not add a newline but it does.

**Comment by sabberworm** [comment]
Good catch. Let’s buffer output when `print` is called and remove the extra blank line.
